schema/pgsql/upgrades/026.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE contact ADD COLUMN external_uuid uuid NOT NULL;
+ALTER TABLE contactgroup ADD COLUMN external_uuid uuid NOT NULL;
+
+ALTER TABLE contact ALTER COLUMN color DROP NOT NULL;
```

Comment:
Removing that column is neither mentioned in the commit message, the PR description, nor the linked issue. If this made it into the PR intentionally, please elaborate.

Reply:
Ahh, there's already a [issue](https://github.com/Icinga/icinga-notifications-web/issues/200) for this, so I'm deleting this change here.